src/components/statistics/progress-bars/Widgets/CircleBars.vue

```diff
@@ -0,0 +1,40 @@
+<template>
+  <vuestic-widget class="progress-bar-widget" :headerText="$t('progressBars.circle')">
+    <div class="va-row">
+      <div v-for="n in 10" :key="n" class="flex sm1 md1">
+        <vuestic-circle-progress :value="value * n / 10"/>
+      </div>
+      <div class="flex sm1 md1">
+        <vuestic-circle-progress :indeterminate="true"/>
+      </div>
+    </div>
+    <div class="va-row">
+      <div v-for="n in 10" :key="n" class="flex sm1 md1">
+        <vuestic-circle-progress :value="value * n / 10" :theme="'Info'">{{ value * n / 10 }}%</vuestic-circle-progress>
+      </div>
+      <div class="flex sm1 md1">
+        <vuestic-circle-progress :indeterminate="true" :theme="'Info'"/>
+      </div>
+    </div>
+  </vuestic-widget>
+</template>
+
+<script>
+export default {
+  data () {
+    return {
+      value: 0
+    }
+  },
+  mounted () {
+    this.animateValue()
+  },
+  methods: {
+    animateValue () {
```

Comment:
Why would we need to make it go 0 -> 100 after timeout? How does animation work?

Reply:
ok